### PR TITLE
[0.53 PowerToy update] - Always on top and new args

### PR DESCRIPTION
**Pull request by crutkas**
Please wait for 0.53 to be released here

### CODE REVIEW

**Comment by PRMerger12** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by PRMerger12** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/always-on-top.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+With the activation / deactivation shortcut (default: <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>T</kbd>), the window will be toggle on top.  By default, you'll have a 
+
+## PowerToys settings
+
+From the settings menu, the following options can be configured:
+
+| Setting | Description |
+| --- | --- |
+| Activation shortcut | The customizable keyboard command to turn on or off mouse highlighting |
```
Comment (Jay-o-Way) [inline hub/powertoys/always-on-top.md]: Copied from MouseUtils? ;)

File: hub/powertoys/index.md [hunk 2]
```diff
@@ -20,13 +20,24 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
...
 
+
+### Always on Top
+
+:::row:::
+    :::column:::
+        [![Always on Top screenshot](../images/pt-always-on-top-menu.png)](always-on-top.md)
+    :::column-end:::
+    :::column span="2":::
+        [Always on Top](always-on-top.md) is a quick and easy way to pin windows on top of of all other windows with a quick key shortcut.
```
Comment (Jay-o-Way) [inline hub/powertoys/index.md]:
```suggestion
        [Always on Top](always-on-top.md) is a quick and easy way to pin windows on top of all other windows with a quick key shortcut.
```

File: hub/powertoys/mouse-utilities.md [hunk 3]
```diff
@@ -37,6 +39,8 @@ From the settings menu, the following options can be configured:
 
 Display visual indicators when the left or right mouse buttons are clicked. By default, mouse highlighting can be turned on and off with the <kbd>Win</kbd> + <kbd>Shift</kbd> + <kbd>H</kbd> shortcut.
 
+### Find my mouse settings
```
Comment (Jay-o-Way) [inline hub/powertoys/mouse-utilities.md]:
```suggestion
### Mouse Highlighter settings
```

File: hub/powertoys/always-on-top.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+# Always on Top utility
+
+A system-wide utility for Windows that allows you to pin windows to the top of your screen. 
+
+![ColorPicker](../images/pt-always-on-top.png)
+
+## Getting started
+
+### Togglign windows to be on top
```
Comment (Jay-o-Way) [inline hub/powertoys/always-on-top.md]:
```suggestion
### Toggling windows to be on top
```

File: hub/powertoys/always-on-top.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+With the activation / deactivation shortcut (default: <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>T</kbd>), the window will be toggle on top.  By default, you'll have a
```
Comment (Jay-o-Way) [inline hub/powertoys/always-on-top.md]: - I thought it was "win + ctrl + T" ? (based on [this commit](https://github.com/microsoft/PowerToys/blob/d2de24472235eddb4b35c5a9112520a8409da0af/src/modules/alwaysontop/AlwaysOnTop/Settings.h))
- sentence is truncated

File: hub/powertoys/always-on-top.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+description: A system-wide utility for Windows that allows you to pin windows to the top of your screen. 
+ms.date: 05/28/2021
+ms.topic: article
+ms.localizationpriority: medium
+no-loc: [PowerToys, Windows, Always on Top]
+---
+
+# Always on Top utility
+
+A system-wide utility for Windows that allows you to pin windows to the top of your screen.
```
Comment (Jay-o-Way) [inline hub/powertoys/always-on-top.md]: "top of screen" normally mean in Y-axis, that's not the case...

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]

**Comment by PRMerger5** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/always-on-top.md (hunk 5 above)
Comment (crutkas) [reply hub/powertoys/always-on-top.md]: you are correct, looks like i had an old setting

**Comment by PRMerger4** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger-2** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/install.md [hunk 7]
```diff
@@ -49,28 +49,31 @@ To install PowerToys using the [Windows Package Manager](../package-manager/wing
 winget install Microsoft.PowerToys --source winget
 ```
 
-## Installer arguments 
+## Installer args:
 
-Our installer has a list of arguments you can chain on to accomplish different tasks such as a silent install.
+The installer execuatable accepts the [Microsoft Standard Installer command-line options](https://docs.microsoft.com/en-us/windows/win32/msi/standard-installer-command-line-options)
```
Comment (jaimecbernardo) [inline hub/powertoys/install.md]: execuatable -> executable

File: hub/powertoys/always-on-top.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+A system-wide utility for Windows that allows you to pin windows to the top of all your windows. 
+
+![ColorPicker](../images/pt-always-on-top.png)
+
+## Getting started
+
+### Toggling windows to be on top
+
+With the activation / deactivation shortcut (default: <kbd>⊞ Win</kbd>+<kbd>Ctrl</kbd>+<kbd>T</kbd>), the window will be toggle on top.
```
Comment (jaimecbernardo) [inline hub/powertoys/always-on-top.md]: "toggle on top" -> "toggled on top"

**Comment by PRMerger12** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by crutkas** [comment]
can we get this merged in?

**Comment by PRMerger3** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger4** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattwojo — APPROVED** [review]
Made a few minor editorial updates

**Pull request merged**